arroyo/backends/kafka/configuration.py

```diff
@@ -70,6 +70,7 @@ def build_kafka_consumer_configuration(
     default_config: Mapping[str, Any],
     group_id: str,
     auto_offset_reset: Optional[str] = None,
+    force_offset_reset: Optional[str] = None,
```

Comment:
Should this be a bool?

Reply:
It used to be a book to "force"t eh value of `auto_offset_reset`, now it just overrides the value of `auto_offset_reset` so it has the same values as `auto_offset_reset`.